src/main/java/com/client/core/base/dao/impl/StandardJpaDao.java

```diff
@@ -201,9 +192,9 @@ private Long getTotal(TypedQuery<T> query) {
         query.getParameters().stream().forEach( parameter -> {
             String name = parameter.getName();
 
-            for(int position : parameterTranslations.getNamedParameterSqlLocations(name)) {
+            for(int position : parameterTranslations.getPositionalParameterInformationMap().keySet()) { // TODO: Find replacement (This could work, but needs testing)
```

Comment:
Kinda worried about this one - changing named parameters for positional parameters kind of a big deal.

Reply:
Still trying to do some testing on this (WIP)